imports/plugins/core/orders/client/containers/orderDashboardContainer.js

```diff
@@ -194,6 +270,468 @@ class OrderDashboardContainer extends Component {
     return false;
   }
 
+  /**
+   * shippingStatusUpdateCall
+   *
+   * @summary set selected order(s) to the provided shipping state
+   * @param {Array} selectedOrders - array of selected orders
+   * @param {String} status - the shipping status to be set
+   * @return {null} no return value
+   */
+  shippingStatusUpdateCall = (selectedOrders, status) => {
+    this.setState({
+      isLoading: {
+        [status]: true
+      }
+    });
+    let orderText = "order";
+
+    if (selectedOrders.length > 1) {
+      orderText = "orders";
+    }
+
+    // capitalize the first letter of the shipping status passed in
+    // e.g. 'shipped' becomes 'Shipped'.
+    // status[0].toUpperCase() capitalizes the first letter of the string
+    // status.substr(1).toLowerCase() converts every other letter to lower case
+    const capitalizeStatus = status[0].toUpperCase() + status.substr(1).toLowerCase();
+    let orderCount = 0;
+
+    // TODO: send these orders in batch as an array. This would entail re-writing the Meteor calls to update the
+    // different shipping statuses to receive an array of objects(orders) as a param
+
+    // TODO: rethink this type of flow for updating shipping statuses
+    selectedOrders.forEach((order) => {
+      Meteor.call(`orders/shipment${capitalizeStatus}`, order, order.shipping[0], (err) => {
+        if (err) {
+          Alerts.toast(`An error occured while setting the status: ${err}`, "error");
+        } else {
+          Meteor.call("orders/updateHistory", order._id, "Shipping state set by bulk operation", status);
+        }
+        orderCount++;
+        if (orderCount === selectedOrders.length) {
+          this.setState({
+            shipping: {
+              [status]: true
+            },
+            isLoading: {
+              [status]: false
+            }
+          });
+          Alerts.alert({
+            text: i18next.t("order.orderSetToState", {
+              orderNumber: selectedOrders.length,
+              orderText: orderText,
+              status: status
+            }),
+            type: "success",
+            allowOutsideClick: false
+          });
+        }
+      });
+    });
+  }
+
+  displayAlert = (selectedOrders, status, options) => {
+    // capitalize the first letter of the shipping status passed in
+    // e.g. 'shipped' becomes 'Shipped'.
+    // status[0].toUpperCase() capitalizes the first letter of the string
+    // status.substr(1).toLowerCase() converts every other letter to lower case
```

Comment:
As an example for why this is important, but also why I'm an idiot, I was thinking that status[0] was the first item in an array during the last review not reading close enough to see that it's actually just pulling the first letter of the string.  Thanks for adding these comments.